pkg/cluster/nodes/nodes.go

```diff
@@ -134,3 +134,49 @@ func WaitForReady(node *Node, until time.Time) bool {
 		return true
 	})
 }
+
+// Restart restarts nodes by name / ID (see Node.String())
+func Restart(nodes ...Node) error {
+	if len(nodes) == 0 {
+		return nil
+	}
+	ids := []string{}
+	for _, node := range nodes {
+		ids = append(ids, node.name)
+	}
+	cmd := exec.Command(
+		"docker",
+		append(
+			[]string{
+				"restart",
+				// the default wait time is 10s
+				"--time=1",
```

Comment:
Is 1s enough wait for a successful restart?

Reply:
to my understanding it's 1 second before killing the container.
https://docs.docker.com/engine/reference/commandline/restart/

Reply:
yep, docker will send `SIGTERM` and after the wait time will send `SIGKILL`. 
set it to `1s` just for speed up.